### PR TITLE
Add various V_CVT opcodes

### PR DESCRIPTION
**Pull request by korenkonder**

### CODE REVIEW

File: src/shader_recompiler/ir/ir_emitter.cpp [hunk 1]
```diff
@@ -1402,6 +1402,15 @@ F16F32F64 IREmitter::FPConvert(size_t result_bitsize, const F16F32F64& value) {
...
             return Inst<F32>(Opcode::ConvertF32F16, value);
+        case Type::F64:
+            return Inst<F32>(Opcode::ConvertF32F64, value);
+        default:
+            break;
+        }
+    case 64:
+        switch (value.Type()) {
+        case Type::F32:
+            return Inst<F16>(Opcode::ConvertF64F32, value);
```
Comment (polybiusproxy) [inline src/shader_recompiler/ir/ir_emitter.cpp]: Shouldn't this be `Inst<F64>`?

Comment (korenkonder) [reply src/shader_recompiler/ir/ir_emitter.cpp]: Oh. Yeah. Kinda forgot that.

**Comment by korenkonder** [comment]
Fixed conflicts with main. Should be good now.

**Comment by Hermiten** [comment]
Do you know any games that use these opcodes ?

**Review by polybiusproxy — APPROVED** [review]

**Comment by korenkonder** [comment]
Fixes https://github.com/shadps4-emu/shadPS4/issues/496#issuecomment-2351441658

**Pull request merged**